web/profiles/webspark/webspark/config/install/editor.editor.basic_html.yml

```diff
@@ -4,63 +4,72 @@ dependencies:
   config:
     - filter.format.basic_html
   module:
-    - ckeditor
+    - ckeditor5
 format: basic_html
-editor: ckeditor
+editor: ckeditor5
 settings:
   toolbar:
-    rows:
-      -
-        -
-          name: Formatting
-          items:
-            - Bold
-            - Italic
-        -
-          name: Linking
-          items:
-            - Anchor
-            - DrupalLink
-            - DrupalUnlink
-        -
-          name: Lists
-          items:
-            - BulletedList
-            - NumberedList
-        -
-          name: Media
-          items:
-            - DrupalFontAwesome
-            - DrupalImage
-            - DrupalMediaLibrary
-        -
-          name: 'Block Formatting'
-          items:
-            - Format
-        -
-          name: Tools
-          items:
-            - Source
-        -
-          name: Webspark
-          items:
-            - AddResponsiveArea
-            - WebsparkButton
-            - WebsparkDivider
-            - HorizontalRule
-            - WebsparkBlockquote
-            - WebsparkHighlightedHeading
-            - WebsparkLead
-        -
-          name: Table
-          items:
-            - WebsparkTable
+    items:
+      - bold
+      - italic
+      - '|'
+      - link
+      - '|'
+      - bulletedList
+      - numberedList
+      - '|'
+      - fontawesome
+      - drupalInsertImage
+      - drupalMedia
+      - '|'
+      - heading
+      - '|'
+      - sourceEditing
+      - '|'
+      - horizontalLine
+      - responsiveArea
+      - websparkButton
+      - websparkDivider
+      - websparkLead
+      - websparkHighlitedHeading
```

Comment:
I've noticed files related to the Highlighted Heading are mostly named `WebsparkHighlightedHeading*` but in the code we use `WebsparkHighlitedHeading*` ... can we make the naming consistent to avoid future confusion?